### ISSUE TITLE
Tests fail because of fixtures

### ISSUE THREAD

**Issue by esquinas** [mongoid/mongoid-demo#20]
On the rails demo, fixtures seem to be incompatible with Mongoid because they come from ActiveRecord. I'd suggest using "factory_bot_rails" or "mongoid_fixtures", they are very different approaches.

Anyway, it is very confusing to follow the docs, then run the tests to see every one of them failing with no explanation.

The error messages themselves are not helpful either:
`NoMethodError: undefined method `posts' for #<PostsControllerTest:0x0000...`

I think it is important to keep demo code updated to help newcomers or people just-trying Mongoid.

Thanks!

**Comment by p-mongo** [comment]
The tests were never written for these applications, which would explain why they aren't working.

We currently use RSpec for testing, as I believe most of the Ruby community does, therefore I removed the generated tests using test/unit for the time being.

**Comment by esquinas** [comment]
Personally, I am new to RSpec, having more experience with Minitest instead, which is the off-the-shelf test suite for Rails 6.
I just managed to have the tests working adding 'factory_bot_rails' gem but it was not straight-forward. I had to change this: `parallelize(workers: 1)` in  test_helper.rb and added ` teardown do; Mongoid.purge!; end` on each test file, apart from the specifics for FactoryBot, of course.

Anyway It would be very nice to see the "standard" (at least, ideally: "recommended") configuration for RSpec + Mongoid along with some green ~~tests~~ specs.

I hope to see those soon. Thanks for the quick response!
